stf/src/commands/common_args_processing.rs

```diff
@@ -58,13 +58,13 @@ pub fn get_order_from_matches(
         price: arg_price,
     };
 
-    return Ok(order);
+    Ok(order)
 }
 
 pub fn get_cancel_order_from_matches(matches: &ArgMatches, main_account: AccountId) -> Result<CancelOrder, String> {
     let order_id = matches
         .value_of(ORDER_UUID_ARG_NAME)
-        .expect(format!("missing {} argument", ORDER_UUID_ARG_NAME).as_str())
+        .unwrap_or_else(|| panic!("missing {} argument", ORDER_UUID_ARG_NAME))
```

Comment:
Does clippy say it is better style to use `unwrap_or_else(|| panic!())` instead of `expect()` ? I'll have to remember that..

Reply:
Yes, it does. Here's the reasoning: https://rust-lang.github.io/rust-clippy/master/index.html#expect_fun_call